Assets/Scripts/ForestController.cs

```diff
@@ -32,38 +35,100 @@ public float Sap
 
     #endregion Public Properties
 
+    #region Private Properties
+
+    private Action SelectedAction
+    {
+        get { return _selectedAction; }
+        set
+        {
+            _selectedAction = value;
+
+            switch (value)
+            {
+                case (Action.Grow):
+                    _selectedActionUI.text = "<=> Grow Branch <=>";
+                    break;
+                case (Action.Destroy):
+                    _selectedActionUI.text = "<=> Destroy Branch <=>";
+                    break;
+            }
+        }
+    }
+
+    private bool ScrollingUp
+    {
+        get { return Input.GetAxis("Mouse ScrollWheel") > 0f; }
```

Comment:
Maybe we can have an alternate option for using arrow keys, like:

get 
{
    return ((Input.GetAxis("Mouse ScrollWheel") > 0f) || (Input.GetKeyDown(KeyCode.UpArrow))
}

Reply:
I implemented this.